modules/lazygit/testbeds/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, pkgs, ... }:
+
+let
+  script = pkgs.writeShellScript "startup" ''
+    ${lib.getExe pkgs.git} init
+    ${lib.getExe pkgs.lazygit}
+  '';
+in
+{
+  stylix.testbed.ui.command = {
+    text = toString script;
```

This seems like unnecessary indirection
```suggestion
{
  stylix.testbed.ui.command = {
    text = ''
      ${lib.getExe pkgs.git} init
      ${lib.getExe pkgs.lazygit}
    '';
```

---

The desktop file generation embeds the raw text. So, I did this to run multiple commands.

This happens if I apply your suggestion:
`error: file contains line "/nix/store/fgfkz9lcrd5bvl7i6bd1i3p3vdsxpjx7-lazygit-0.49.0/bin/lazygit", which is not a comment, a group or an entry`

---

try
```
text = "${lib.getExe pkgs.git} init && ${lib.getExe pkgs.lazygit}"
```

---

Also doesn't work, `&` is reserved. So is `;`. Quoting doesn't work as well.

---

what do you mean it's reserved? it's inside a string..

---

Sorry, reserved by the .desktop format.

---

ohh, well then inlining "script" would be good